airbyte-integrations/connectors/source-mixpanel/source_mixpanel/source.py

```diff
@@ -84,7 +114,7 @@ def check_connection(self, logger: AirbyteLogger, config: Mapping[str, Any]) ->
             config = self._validate_and_transform(config)
             auth = self.get_authenticator(config)
         except Exception as e:
-            return False, e
+            return False, e.message
 
         # https://github.com/airbytehq/airbyte/pull/27252#discussion_r1228356872
         # temporary solution, testing access for all streams to avoid 402 error
```

Comment:
This is workaround for this issue: https://github.com/airbytehq/airbyte/pull/27252#discussion_r1228356872

Reply:
Right, but the comment says it is testing access for all streams, but it is only testing access for one stream?

Reply:
Notably, it used to `break` if successfully connected to one stream, but now returns True: 

https://github.com/airbytehq/airbyte/pull/27252/files#diff-ec2bcf88395f3b6aa5b5016d6d83b6b0c0270640e6cbb699c04034b9c9c4e0d4R95

That doesn't look like it was introduced in this PR, so I won't be a stickler about it, but a follow up issue would be good

Reply:
The snippet from your comment behaves similarly but breaks instead of returning. The comment indicates that the system tests to see if any stream functions correctly. If at least one does, the check is deemed successful. I'll rephrase it for better clarity.

Reply:
Ah, if thats the case, then yes I've misinterpreted the comment - thanks

I guess I was thinking `continue` (onto the next one), not `break`